src/main/java/Duke.java

```diff
@@ -1,10 +1,32 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?\n\n");
+
+        int taskCounter = 0;
+        String line;
+        String[] tasks = new String[100];
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
```

Comment:
I like your variable names, they clearly state the use of the variable which increases your code readability!